valhalla/baldr/double_bucket_queue.h

```diff
@@ -163,7 +192,7 @@ template <typename label_t> class DoubleBucketQueue final {
   bucket_t overflowbucket_;
 
   // Access to a container of labels to get cost given the label index.
-  const std::vector<label_t>& labelcontainer_;
+  const std::vector<label_t>* labelcontainer_;
```

Comment:
why change this?

Reply:
ahh that makes sense. its really hard to see a detail like that without it being spelled out